test/jdk/java/awt/Graphics/LCDTextAndGraphicsState.java

```diff
@@ -79,14 +107,81 @@ public void test3(Graphics g) {
     }
 
     public Dimension getPreferredSize() {
-        return new Dimension(500,600);
+        return new Dimension(500, 600);
     }
 
-    public static void main(String[] args) throws Exception {
+    private static void createInstructionUI() {
+        GridBagLayout layout = new GridBagLayout();
+        Panel mainControlPanel = new Panel(layout);
+        Panel resultButtonPanel = new Panel(layout);
+        GridBagConstraints gbc = new GridBagConstraints();
+        gbc.gridx = 0;
+        gbc.gridy = 0;
+        gbc.insets = new Insets(5, 15, 5, 15);
+        gbc.fill = GridBagConstraints.HORIZONTAL;
+
+        TextArea instructionTextArea = new TextArea();
+        instructionTextArea.setText(INSTRUCTIONS);
+        instructionTextArea.setEditable(false);
+        instructionTextArea.setBackground(Color.white);
```

Comment:
Is there a particular reason to set the background as white? It is better if the background and foreground colors are set by the L&F itself.  I have fixed couple of bugs for GTKL&F, where the panel background color was set manually and then the L&F chosen color for foreground/Text was also same, so the text was not visible at all.

Reply:
Nice point about look and feel. Sure, let me remove the background color.